### DISCUSSION TITLE
Panoptic segmentation still not supported?

### DISCUSSION THREAD

**Discussion by mvdelt** [cvat-ai/cvat#2878] [Q&A] [answered]
issue #1143, issue #1784 says Panoptic segmentation will be supported soon.
But still can't find any docs in the user's guide.
Is it already supported, or it's on the way?

If it's not supported yet, how can we make one in COCO or Cityscapes format?

Thanks!

**Comment by azhavoro** [comment, marked as answer]
@zhiltsov-max Could you please answer?

**Reply by zhiltsov-max** [reply]
@mvdelt, it is not implemented yet. We have these formats in our plans, but they are not scheduled for implementation. If you want to contribute, consider extending [`COCO` format in Datumaro](https://github.com/openvinotoolkit/datumaro/tree/develop/datumaro/plugins/coco_format).

**Reply by zhiltsov-max** [reply]
It should already be available in the `COCO` format upload. The dataset layout expected in the archive is described here: https://openvinotoolkit.github.io/datumaro/docs/formats/coco/#import-coco-dataset (images are not needed).
